cypress/integration/Regression/CreateModels/ApiCallbacks.spec.js

```diff
@@ -20,11 +20,11 @@ describe.skip('API Callbacks - CreateModels', () => {
 
   context('Create Entities', () => {
     it('Should create custom trained entities that will be used by the API Callbacks', () => {
-      entities.CreateNewEntity({ type: 'Custom Trained', name: '1stArg'})
-      entities.CreateNewEntity({ type: 'Custom Trained', name: '2ndArg'})
-      entities.CreateNewEntity({ type: 'Custom Trained', name: 'entityError', negatable: true})
-      entities.CreateNewEntity({ type: 'Custom Trained', name: 'logicError', negatable: true})
-      entities.CreateNewEntity({ type: 'Custom Trained', name: 'renderError', negatable: true})
+      entities.CreateNewEntityThenVerifyInGrid({ type: 'Custom Trained', name: '1stArg'})
```

Comment:
I would mention that expanding the function names like this is a sign of the function doing too many things. Ideally want atomic / independent things that compose together.  Example, you would also want to be able to verify grid even after an Update instead of Create, or be to create an entity within an Action and then go to entity page and verify it's in the grid correctly which injects navigation steps.  All these variations are not allowed when coded like this.

If you're going to keep doing this at least make the functions internally separate.
You have `CreateNewEntityThenVerifyInGrid` which calls `CreateNewEntity` but still has the verifying grid logic directly inside it on the same level.  Not important for this PR but in future I'd rather see:

- CreateNewEntityThenVerifyInGrid
  - CreateNewEntity
  - VerifyEntityInGrid

Reply:
Okay, I'll do that soon.